### PR TITLE
Fixes fmt & typos

### PR DESCRIPTION
**Pull request by Bwko**

### CODE REVIEW

File: remote/github/client.go [hunk 1]
```diff
@@ -36,8 +36,8 @@ func NewClient(uri string) *Client {
 // authenticates all outbound requests with the given token.
 func NewClientToken(uri, token string) *Client {
 	config := new(oauth2.Config)
-	auther := config.Client(oauth2.NoContext, &oauth2.Token{AccessToken: token})
-	return &Client{auther, uri}
+	author := config.Client(oauth2.NoContext, &oauth2.Token{AccessToken: token})
```
Comment (tboerger) [inline remote/github/client.go]: This is not a typo, it's the auther and not author.

Comment (lunny) [reply remote/github/client.go]: Really. :)

**Comment by Bwko** [comment]
It was incorrectly marked by misspell as a typo. Changed it back to auther :)

**Comment by tboerger** [comment]
LGTM

**Comment by lunny** [comment]
LGTM

**Pull request merged**